packages/components/src/client/styles/tag.scss

```diff
@@ -1,105 +1,111 @@
+// Developer beware: this file is symlinked between components/ and theme/
+
 @use "sass:color";
 @use "sass:list";
 
 .tags-wrapper {
   list-style: none;
   margin: 0;
   padding-left: 0;
+  .tag-item {
+    display: inline-block;
+    margin-right: 0.5rem;
+    vertical-align: middle;
+    overflow: hidden;
+  }
 
-  .tag {
-    position: relative;
+  /*.tag-item */ .tag {
+  display: inline-block;
 
-    min-width: 1.5rem;
```

Comment:
Do not try to delete styles unless you make sure you understand why was that style.

A tag with single letter a should not look like `a`, it's werid, and also, all clickable element should be bigger than a minimal size(for button it's 48*48, for links it should be at list 1.5em*1.5em), otherwise it obey a11y best praticed.

So this is forcing it to be someting like ``　a　``.

The other styles are probably similar.